Sofa/Component/IO/Mesh/src/sofa/component/io/mesh/VoxelGridLoader.cpp

```diff
@@ -46,7 +46,7 @@ VoxelGridLoader::VoxelGridLoader()
       roi ( initData ( &roi, Vec6i ( 0,0,0, 0xFFFF, 0xFFFF, 0xFFFF ), "ROI", "Region of interest (xmin, ymin, zmin, xmax, ymax, zmax)" ) ),
       headerSize ( initData ( &headerSize, 0, "header", "Header size in bytes" ) ),
       segmentationHeaderSize ( initData ( &segmentationHeaderSize, 0, "segmentationHeader", "Header size in bytes" ) ),
-      idxInRegularGrid(initData(&idxInRegularGrid,"idxInRegularGrid","indices of the hexa in the grid.")),
+      idxInRegularGrid(initData(&idxInRegularGrid,"d_idxInRegularGrid","indices of the hexa in the grid.")),
```

Comment:
```suggestion
      idxInRegularGrid(initData(&idxInRegularGrid,"idxInRegularGrid","indices of the hexa in the grid.")),
```
and you did not renamed the Data...